src/server/service/kv.rs

```diff
@@ -666,9 +690,31 @@ impl<T: RaftStoreRouter<RocksEngine> + 'static, E: Engine, L: LockManager> Tikv
                             my_store_id: store_id,
                         }));
                     }
-                    if let Err(e) = ch.send_raft_msg(msg) {
-                        return future::err(Error::from(e));
+                    let mut flag = true;
+                    let disk_full_precheck = || {
+                        if cfg!(feature = "failpoints") {
+                            fail_point!("disk_full_follower_1", store_id == 1, |_| {
```

Comment:
It's possible that store 1 is leader. Actually for lots of test cases, store 1 is the leader.

Reply:
And you need to move the check into raftstore, before `step`.

Reply:
> It's possible that store 1 is leader. Actually for lots of test cases, store 1 is the leader.
1. I have set the leader not 1 and 2.

> And you need to move the check into raftstore, before `step`.
1. to filter in batch_raft will save all the msg process resourse.
2. So, my problem is why we should move to step?